### PR TITLE
updated READMEs with more details.

### PR DESCRIPTION
**Pull request by mol3earth**
- added some install instructions for the github repo and the conda package to the README
- fixed some links in the README
- added a simple Table of Contets README to the docs/reference/catalog/ 
edit:
- now includes working environment.yml file

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -1,10 +1,149 @@
...
+  - usgs-astrogeology
+dependencies:
+  - _libgcc_mutex=0.1
+  - _openmp_mutex=4.5
+  - affine=2.3.0
+  - attrs=19.3.0
+  - boost-cpp=1.72.0
+  - brotlipy=0.7.0
+  - bzip2=1.0.8
+  - ca-certificates=2020.6.20
```
Comment (Kelvinrr) [inline environment.yml:17]: I believe you have a more up to date version, mind updating this PR with the new env file?

Comment (Kelvinrr) [reply environment.yml]: I think just remove the env file from the changes as this was handled in a different PR.

Comment (mol3earth) [reply environment.yml]: how's that

Comment (Kelvinrr) [reply environment.yml]: Awesome, thanks 👍

**Pull request merged**